playbooks/Installation/expose_docker_registry.adoc

```diff
@@ -0,0 +1,149 @@
+= OpenShift Enterprise 3 Exposing Internal Docker Registry
+Andrew Block <ablock@redhat.com>
+v1.0, 2015-10-02
+:scripts_repo: https://github.com/rhtconsulting/rhc-ose
+:toc: macro
+:toc-title:
+
+toc::[]
+
+OpenShift provides an internal Docker registry for which to serve images for use within the OpenShift environment. It may be desirable to expose this registry for consumption by external entities. This document describes the process of accessing the integrated Docker registry from a source external to the OpenShift environment.
+
+== Prerequisites
+
+The following prerequisites are required prior to beginning the steps to expose the internal docker repository
```

Comment:
At minimum, this document needs to reference the official documentation: https://docs.openshift.com/enterprise/3.0/install_config/install/docker_registry.html#exposing-the-registry

I think there's also a potential to trim this doc, as some of the items covered here have since been added to the official docs.